packages/page-staking/src/Nomination/index.tsx

```diff
@@ -0,0 +1,380 @@
+    setAlreadyHaveStashes(true);
+    setStepsState(['completed', 'completed', 'completed', 'completed']);
+  }, [ownStashes]); */
+
+  return (
+    <main className={`${className} ${!isVisible ? 'staking--hidden' : ''} simple-nominatio`}>
+      <TabsHeader
+        setCurrentStep={setCurrentStep}
+        stepsState={stepsState}
+        steps={steps}
+        currentStep={currentStep}
+      />
+      <div className="ui attached segment">
+        {/* {feesLoading && (
+          <Spinner />
+        )} */}
+        {currentStep === steps[0] &&
+        <>
+            <br />
+            <h3>{t('Select your account that holds funds')}:</h3>
+            <br />
+            <Available label={balanceWrapper(t('Your account balance'))} params={senderId} />
+            <AccountSelector
+              value={senderId}
+              title={'Your account'}
+              onChange={setSenderId}
+              stepsState={stepsState}
+              setStepsState={setStepsState}
+            />
+        </>
+        }
+        {currentStep === steps[1] &&
+        <>
+            <br />
+            <h3>{t('Now you need to select or create Controller account. This is account that will manage your funds')}:</h3>
+            <br />
+            <Available label={balanceWrapper('Controller balance')} params={controllerAccountId} />
+            <ControllerAccountSelector
+              senderId={senderId}
+              value={controllerAccountId}
+              title={t('controller account')}
+              onChange={resetControllerInfo}
+              stepsState={stepsState}
+              setStepsState={setStepsState}
+              toggleCreate={toggleCreate}
+            />
+          {isCreateOpen && (
+            <CreateModal
+              hideAdvanced
+              initialName={'NominationController'}
+              onClose={toggleCreate}
+              onStatusChange={onStatusChange}
+            />
+          )}
+        </>
+        }
+        {currentStep === steps[2] &&
+        <>
+          <br />
+          {!isBalanceEnough() &&
+          <h3>
+            {t('Now we will transfer some small amount from your account that holds funds to Controller so that it can pay transaction fees. Just click Next to proceed.')}
+          </h3>
+          }
+          { !controllerAlreadyBonded && isBalanceEnough() && (
+              <>
+                <h3>{t('Now we need to Bond funds. Bonding means that main account gives control over funds to Controller account.')}
+                  <p>{t('Once bonded, funds will be under management of your Controller.')}</p>
+                  <p>{t('Money can be unbonded, but will remain locked for a while.')}</p>
+                </h3>
+                <h4 className="ui orange header">
+                  {t('Warning: After bonding, your funds will be locked and will remain locked after the nomination is stopped for')} <EraToTime showBlocks />, {t('which is approximately')} <EraToTime showDays />.
```

Comment:
This approach to i18n does not quite work. Use the `Trans` component, you are assuming all languages use the same constructs here.